source/_integrations/zwave_js.markdown

```diff
@@ -228,6 +228,41 @@ It's recommended to create a backup before making any major changes to your Z-Wa
    - **Result**: The backup file is downloaded to the device from which you initiated the download.
 4. Done! Store the backup file somewhere safe in case you need it later to restore your Z-Wave network.
 
+## Updating the firmware of your Z-Wave device
+
+Controllers and devices with the Firmware Update Metadata Command Class allow you to update the firmware by uploading a firmware file. In those cases, you can start the firmware update from the device page in Home Assistant. Refer to the documentation of the device manufacturer to find the corresponding firmware file. An example is the [firmware page by Zooz](https://www.support.getzooz.com/kb/article/1158-zooz-ota-firmware-files/).
+
+{% caution %}
+**Risk of damage to the device due to firmware update**
+
+A firmware update can damage your Z-Wave device.
+
+- Before updating your Z-Wave device, make sure an update is necessary, and that you have the correct firmware file matching your device.
+- Once you have started the update process, you must not interrupt the update process but let it complete.
+
+The Home Assistant and Z-Wave JS teams do not take any responsibility for any damages to your device as a result of the firmware update and will not be able to help you if you render your device useless due to firmware update.
+{% endcaution %}
+
+### Prerequisites
+
+- Administrator rights in Home Assistant
+- Downloaded the firmware file from the manufacturer website
+
+### To update firmware of a Z-Wave device
+
+1. In Home Assistant, go to {% my integrations title="**Settings** > **Devices & services**" %}.
+2. Select the **Z-Wave** integration. Then, select **Configure** and select the controller.
+3. Under **Device info**, select the three dots {% icon "mdi:dots-vertical" %} menu, then select **Update firmware**.
```

Comment:
This is outdated and the image is outdated. The update button is now found to the left of the three dots menu.

Reply:
I'll double check what the current state is. It should have been released already I think.

Reply:
So the screen shot you had will be correct for the next release.

Reply:
thank you for checking! rephrased.